discord/ext/commands/bot.py

```diff
@@ -295,11 +299,18 @@ def whitelist(ctx):
             Whether the user is the owner.
         """
 
-        if self.owner_id is None:
+        if self.owner_id:
+            return user.id == self.owner_id
+        elif isinstance(self.owner_ids, (list, tuple, set)):
```

Comment:
You seem to have misunderstood my previous review.

I said to do this *at* `__init__` time so the performance of *this* function is not impacted. It's an invariant when a *user passes a parameter* so it should be checked for correctness.